.github/workflows/basic_webserver_build_release.yml

```diff
@@ -0,0 +1,164 @@
+on:
+#  pull_request:
+  workflow_dispatch:
+
+# this cancels workflows currently in progress if you start a new one
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+  
+env:
+  # use .tar.gz for quick testing
+  ARCHIVE_FORMAT: .tar.br
+  BASIC_WEBSERVER_BRANCH: main
+
+jobs:
+  fetch-releases:
+    runs-on: [ubuntu-20.04]
+    steps:
+      - uses: actions/checkout@v3
+
+      - run: curl -fOL https://github.com/roc-lang/roc/releases/download/nightly/roc_nightly-linux_x86_64-latest.tar.gz
+      - run: curl -fOL https://github.com/roc-lang/roc/releases/download/nightly/roc_nightly-linux_arm64-latest.tar.gz
+      - run: curl -fOL https://github.com/roc-lang/roc/releases/download/nightly/roc_nightly-macos_x86_64-latest.tar.gz
+      - run: curl -fOL https://github.com/roc-lang/roc/releases/download/nightly/roc_nightly-macos_apple_silicon-latest.tar.gz
+
+      - name: Save roc_nightly archives
+        uses: actions/upload-artifact@v3
+        with:
+          path:  roc_nightly-*
+
+  build-linux-x86_64-files:
+    runs-on: [ubuntu-20.04]
+    needs: [fetch-releases]
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Download the previously uploaded roc_nightly archives
+        uses: actions/download-artifact@v3
+
+      - name: build basic-webserver with surgical linker and also with legacy linker
```

Comment:
It only seems to do it with legacy linker...
Is that a mistake in the name or the implementation?